packages/css/src/checkbox/checkbox.scss

```
@@ -0,0 +1,145 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright (c) 2023 Gemeente Amsterdam
+ */
+
+@import "../../utils/breakpoint";
+
+.amsterdam-checkbox__input {
+  appearance: none;
+  margin-block: 0;
+  margin-inline: 0;
+  opacity: 0%; /* to hide native input field in older iOS */
+
+  &:focus-visible + label {
+    outline: auto;
+    outline-offset: var(--amsterdam-checkbox-outline-offset);
+  }
+}
+
+.amsterdam-checkbox__label {
+  color: var(--amsterdam-checkbox-color);
+  cursor: pointer;
+  display: inline-flex;
+  font-family: var(--amsterdam-checkbox-font-family);
+  font-size: var(--amsterdam-checkbox-narrow-font-size);
+  font-weight: 400;
+  gap: 0.5rem;
+  line-height: var(--amsterdam-checkbox-line-height);
+  padding-block: 0.5rem;
+
+  @media screen and (width > $amsterdam-breakpoint) {
+    font-size: var(--amsterdam-checkbox-wide-font-size);
+  }
+}
+
+.amsterdam-checkbox__input:disabled + .amsterdam-checkbox__label {
+  color: var(--amsterdam-checkbox-disabled-color);
+  cursor: not-allowed;
+}
+
+.amsterdam-checkbox__input:focus-visible + .amsterdam-checkbox__label,
+:not(.amsterdam-checkbox__input:disabled) + .amsterdam-checkbox__label:hover {
+  color: var(--amsterdam-checkbox-hover-color);
+  text-decoration: underline;
+  text-underline-offset: 3px;
+}
+
+.amsterdam-checkbox__label__checkmark {
```

The double underscore is never nested in BEM, or at least I've never seen it. Usualy only one level, so something like

```suggestion
.amsterdam-checkbox__label-checkmark {
```

or

```suggestion
.amsterdam-checkbox__checkmark {
```

See: https://cssguidelin.es/#bem-like-naming